### DISCUSSION TITLE
Test for Safari?

### DISCUSSION THREAD

**Discussion by msqr1** [emscripten-core/emscripten#21953] [Ideas]
We have test for Chrome and Firefox, but not Safari, another very popular browser?

**Comment by sbc100** [comment]
Adding some safari testing would be great.  I don't know much about macOS browser testing myself but if anyone has some time to look into it it would be great to add to our circleci config.    For chrome and firefix we currently test in "headless" mode on linux, my guess is that that might be necessary for safari but I don't know for sure.

BTW, we run some tests in javascriptcore (jsc) so we do have partial coverage, at least for the JS and wasm engine parts.
